aggregator/src/main/scala/ai/chronon/aggregator/base/SimpleAggregators.scala

```diff
@@ -160,6 +160,82 @@ class Average extends SimpleAggregator[Double, Array[Any], Double] {
   override def isDeletable: Boolean = true
 }
 
+class RunningAverage extends SimpleAggregator[Double, Array[Any], Double] {
```

Comment:
Instead of a new operator would it make sense to add an argument `prevent_overflow` or `running_average` to the Average operator that defaults to false?